### PR TITLE
Created basic CRUD for courses

### PR DESCRIPTION
**Pull request by MikolaLvivUA**

### CODE REVIEW

File: src/controllers/course/course.controller.ts [hunk 1]
```diff
@@ -9,44 +9,74 @@ import { courseValidator } from '../../validators';
...
+      const { course_id } = req.params;
 
-            if (courseValidity.error) {
-                return next( new ErrorHandler(ResponseStatusCodesEnum.BAD_REQUEST, courseValidity.error.details[0].message));
-            }
+      const gettingCourse = await courseService.getCourseByID(course_id);
 
-            await courseService.createCourse(course);
+      if (!gettingCourse) {
+          return next(new ErrorHandler(ResponseStatusCodesEnum.NOT_FOUND, 'Course is not found', 404));
```
Comment (VictorKmin) [inline src/controllers/course/course.controller.ts]: add custorm error code

File: src/controllers/course/course.controller.ts [hunk 2]
```diff
@@ -9,44 +9,74 @@ import { courseValidator } from '../../validators';
...
-    editCourse(req: IRequestExtended, res: Response, next: NextFunction) {
-        try {
-            const { course_id } = req.params;
+      await courseService.deleteCourseByID(course_id);
 
-            res.json(`${course_id} edited (NO)`);
-        } catch (e) {
-            next(e);
-        }
+      res.status(ResponseStatusCodesEnum.CREATED).json(`course ${course_id} has been deleted`);
```
Comment (VictorKmin) [inline src/controllers/course/course.controller.ts]: ResponseStatusCodesEnum.CREATED ?

File: src/controllers/course/course.controller.ts (hunk 1 above)
Comment (MikolaLvivUA) [reply src/controllers/course/course.controller.ts]: fixed

File: src/controllers/course/course.controller.ts (hunk 2 above)
Comment (MikolaLvivUA) [reply src/controllers/course/course.controller.ts]: fixed

File: src/constants/enums/response-status-codes.enum.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 export enum ResponseStatusCodesEnum {
+    OK = 200,
```
Comment (VictorKmin) [inline src/constants/enums/response-status-codes.enum.ts]: res.end and res.json automatically send 200 status

File: src/controllers/course/course.controller.ts [hunk 4]
```diff
@@ -2,51 +2,85 @@ import { NextFunction, Response } from 'express';
...
-    editCourse(req: IRequestExtended, res: Response, next: NextFunction) {
-        try {
-            const { course_id } = req.params;
+      await courseService.deleteCourseByID(course_id);
 
-            res.json(`${course_id} edited (NO)`);
-        } catch (e) {
-            next(e);
-        }
+      res.status(ResponseStatusCodesEnum.OK).json(`course ${course_id} has been deleted`);
```
Comment (VictorKmin) [inline src/controllers/course/course.controller.ts]: res.end will be enough

**Pull request merged**